src/domains/collections.ts

```diff
@@ -186,105 +213,116 @@ export const collectionToStac = (collection: any): STACCollection => {
     },
   ];
 
+  const provider = [
+    {
+      name: collection.conceptId?.split("-")[1] ?? "CMR",
```

Comment:
I don't think this is a problem since there is validation in CMR against provider names but, I'll go check. if you have a provider with a name like: `provider-1-something`
then do you run into the issue where your split is going only going get provider from a provider with a longer name.
<Concept-id>-provider-1-something